README.md

```diff
@@ -43,6 +43,13 @@ repos on GitHub, it is still a useful source of relative usage data.
   $ pip install -r requirements.txt
   ```
 
+  To update the `requirements.txt` file to the newest versions of dependencies, run:
+
+  ```sh
+  $ pip install -U pip-tools  # Make sure you have pip-compile installed
+  $ pip-compile --upgrade
+  ```
+
```

Comment:
Interesting! Your blog post makes a very compelling case. But is it strictly nessesary to include this documentation here? This project already has a rather laborious setup process and I'd hate to make it look any more involved than it actually is.

Maybe we could move this to a `Makefile` command?